lib/rops/builder.js

```diff
@@ -0,0 +1,270 @@
+const through = require('through2');
+const csv = require('csv-stringify');
+const archiver = require('archiver');
+const { pick, flatten } = require('lodash');
+
+const proceduresColumns = [
+  { key: 'ropId', header: 'return_id' },
+  { key: 'id', header: 'id' },
+  { key: 'species', header: 'species' },
+  { key: 'otherSpecies', header: 'other_species' },
+  { key: 'severityNum', header: 'no_of_procedures' },
+  { key: 'reuse', header: 'reuse' },
+  { key: 'placesOfBirth', header: 'place_of_birth' },
+  { key: 'nhpsOrigin', header: 'nhp_place_of_birth' },
+  { key: 'nhpsColonyStatus', header: 'nhp_colony_status' },
+  { key: 'nhpsGeneration', header: 'nhp_generation' },
+  { key: 'ga', header: 'genetic_status' },
+  { key: 'newGeneticLine', header: 'creation_of_new_genetic_line' },
+  { key: 'purposes', header: 'purpose' },
+  { key: 'subPurpose', header: 'sub_purpose' },
+  { key: 'subPurposeOther', header: 'sub_purpose_other' },
+  { key: 'regulatoryLegislation', header: 'testing_by_legislation' },
+  { key: 'legislationOther', header: 'legislation_other' },
+  { key: 'regulatoryLegislationOrigin', header: 'legislative_requirements' },
+  { key: 'specialTechnique', header: 'technique_of_special_interest' },
+  { key: 'severity', header: 'actual_severity' },
+  { key: 'severityHoNote', header: 'comments_for_ho' },
+  { key: 'severityPersonalNote', header: 'comments_for_personal_use' },
+  { key: 'licenceNumber', header: 'ppl_number' }
+];
+
+const returnsColumns = [
+  'id',
+  'licenceNumber',
+  'establishmentId',
+  'title',
+  'projectStatus',
+  'issueDate',
+  'expiryDate',
+  'revocation_date',
```

Comment:
Was this one just missed?
```suggestion
  'revocationDate',
```